js/views/settingsVw.js

```diff
@@ -113,7 +114,19 @@ module.exports = Backbone.View.extend({
       self.delegateEvents(); //delegate again for re-render
       self.setFormValues();
       self.setState(self.options.state);
-      //self.renderBlocked(self.model.get("user").blocked);
+      
+      // Since the Blocked Users View kicks off many server calls (one
+      // for each blocked user) and since we are re-rendering the entire
+      // settings view often (after each save), we will cache the Blocked
+      // Users View. If for some reason, you really need to re-render it,
+      // call renderBlocked() explicitly after calling render().
```

Comment:
I think an additional optimization we should consider at some point is lazy loading the blocked users as you scroll. AFAIK, there could potentially be many (dozens, hundreds, thousand???) of blocked users. Since we are making a server call for each one, we would want to control the quantities we load at a given time.